cabal-install/Distribution/Client/CmdInstall.hs

```diff
@@ -573,6 +573,7 @@ installAction (configFlags, configExFlags, installFlags, haddockFlags, newInstal
     globalConfigFlag = projectConfigConfigFile (projectConfigShared cliConfig)
     overwritePolicy = fromFlagOrDefault NeverOverwrite
                         $ ninstOverwritePolicy newInstallFlags
+    dryRun = fromFlagOrDefault False $ installDryRun installFlags
```

Comment:
I just found out about buildSettingDryRun, which is probably better than this. Fixing it.